projects/hslayers/components/add-data/add-data.component.ts

```diff
@@ -59,23 +57,11 @@ export class HsAddDataComponent
       .subscribe((e) => {
         let error = e.toString();
         if (error?.includes('Unsuccessful OAuth2')) {
-          error = this.hsLanguageService.getTranslationIgnoreNonExisting(
-            'COMMON',
-            'Authentication failed. Login to the catalogue.',
-            undefined,
-          );
+          error = 'COMMON.Authentication failed. Login to the catalogue.';
```

Comment:
Hmm.. 🤔 Will this work?

Reply:
Surely https://github.com/hslayers/hslayers-ng/blob/6d1b1a2ceae179eaf8e7ebb06160ed581d722642/projects/hslayers/src/assets/locales/en.json#L170.

But I agree that this translation string is not the best. Needlessly long string

Reply:
I was trying hard to induce this error and test it but I kinda fail to create the necessary circumstances under this error is raised 🤔 So I am going to trust you :-)
It is surprising to me that the string with multiple dots is properly parsed and a relevant key in the JSON object is found.

Reply:
![csw](https://github.com/hslayers/hslayers-ng/assets/44566616/806fdd25-80a9-4587-8dac-469c5bab1238)

Reply:
It's like a magic! Thanks :-)